src/core/network/qgshttpheaders.cpp

```diff
@@ -35,7 +35,7 @@ const QString QgsHttpHeaders::KEY_REFERER = "referer";
 
 QgsHttpHeaders::QgsHttpHeaders() = default;
 
-QgsHttpHeaders::QgsHttpHeaders( const QMap<QString, QVariant> &headers )
+QgsHttpHeaders::QgsHttpHeaders( const QVariantMap &headers )
```

Comment:
This constitutes an API break?

Reply:
no

<img width="395" alt="image" src="https://user-images.githubusercontent.com/127259/210095452-b274bb1d-0e83-4073-af53-52cb86d4551c.png">

Reply:
oops, misread that it was `QMap<QString, QString>`, sorry